test/data/stages/authselect/diff.json

```diff
@@ -0,0 +1,36 @@
+{
+  "added_files": [
+    "/etc/authselect/authselect.conf",
+    "/etc/authselect/dconf-db",
+    "/etc/authselect/dconf-locks",
+    "/etc/authselect/fingerprint-auth",
+    "/etc/authselect/nsswitch.conf",
+    "/etc/authselect/password-auth",
+    "/etc/authselect/postlogin",
+    "/etc/authselect/smartcard-auth",
+    "/etc/authselect/system-auth",
+    "/etc/dconf",
+    "/etc/dconf/db",
+    "/etc/dconf/db/distro.d",
+    "/etc/dconf/db/distro.d/20-authselect",
+    "/etc/dconf/db/distro.d/locks",
+    "/etc/dconf/db/distro.d/locks/20-authselect",
+    "/var/lib/authselect/dconf-db",
+    "/var/lib/authselect/dconf-locks",
+    "/var/lib/authselect/fingerprint-auth",
+    "/var/lib/authselect/nsswitch.conf",
+    "/var/lib/authselect/password-auth",
+    "/var/lib/authselect/postlogin",
+    "/var/lib/authselect/smartcard-auth",
+    "/var/lib/authselect/system-auth"
+  ],
+  "deleted_files": [],
+  "differences": {
+    "/etc/nsswitch.conf": {"mode": [33188, 41471]},
+    "/etc/pam.d/fingerprint-auth": {"mode": [33188, 41471]},
+    "/etc/pam.d/password-auth": {"mode": [33188, 41471]},
+    "/etc/pam.d/postlogin": {"mode": [33188, 41471]},
+    "/etc/pam.d/smartcard-auth": {"mode": [33188, 41471]},
+    "/etc/pam.d/system-auth": {"mode": [33188, 41471]}
```

Comment:
What is up with all those mode changes?

Reply:
I can double check, but `authselect` touches many configuration files when setting a profile.

Reply:
So all of these files are changed to symlinks pointing to `/etc/authselect/` directory after `authselect` is called. The target file has the same mode:
```
$ file ~/devel/osbuild/store/objects/d3744750c9a972321d4ee421085af0fc834ad715d56f34545ba5afa714223366/etc/nsswitch.conf
/home/thozza/devel/osbuild/store/objects/d3744750c9a972321d4ee421085af0fc834ad715d56f34545ba5afa714223366/etc/nsswitch.conf: ASCII text

$ stat --format=%f ~/devel/osbuild/store/objects/d3744750c9a972321d4ee421085af0fc834ad715d56f34545ba5afa714223366/etc/nsswitch.conf
81a4

$ file ~/devel/osbuild/store/objects/82cafde32e9a4eae6e4c7231add3a5f61bab1466f854ee2852d56115d48da9c7/etc/nsswitch.conf
/home/thozza/devel/osbuild/store/objects/82cafde32e9a4eae6e4c7231add3a5f61bab1466f854ee2852d56115d48da9c7/etc/nsswitch.conf: symbolic link to /etc/authselect/nsswitch.conf

$ stat --format=%f ~/devel/osbuild/store/objects/82cafde32e9a4eae6e4c7231add3a5f61bab1466f854ee2852d56115d48da9c7/etc/nsswitch.conf
a1ff

$ stat --format=%f ~/devel/osbuild/store/objects/82cafde32e9a4eae6e4c7231add3a5f61bab1466f854ee2852d56115d48da9c7/etc/authselect/nsswitch.conf
81a4
```